squash/tests/test_api.py

```diff
@@ -0,0 +1,48 @@
+"""Tests for the dashboard API"""
```

Comment:
If I  run py.test in the command line the tests/test_api.py is found and the tests are executed, but it requires the application to be running which is not good. I probably should switch to django test framework then a database would be created for the tests and I dont have to make assumptions about the password which is manually set when the development database is created  (see run.sh script)